apps/nextjs/src/app/api/fetch-owa-lesson/lessonTransformer.ts

```diff
@@ -22,19 +23,24 @@ type OwaLesson = z.infer<typeof lessonContentSchema>;
 /**
  * Transforms Oak's quiz format to the format expected by LessonPlanSchemaWhilstStreaming
  */
-export function transformQuiz(quiz: OwaQuizQuestion[]): QuizV1Question[] {
+export function transformQuiz(quiz: OwaQuizQuestion[]): QuizV2 {
   const quizData = z.array(quizQuestionSchema).parse(quiz);
   if (!quizData || !Array.isArray(quizData)) {
-    return [];
+    return {
+      version: "v2",
+      questions: [],
+    };
   }
 
-  return quizData.map((question) => {
+  const questions: QuizV2Question[] = quizData.map((question) => {
     if (!question || !question.question_stem) {
       return {
+        questionType: "multiple-choice",
         question: "",
         answers: [],
         distractors: [],
-      };
+        imageAttributions: [],
+      } as QuizV2Question;
```

Comment:
I think we don't need these assertions, tsc still happy

Reply:
Yes you're right